static/locales/en-US.yaml

```diff
@@ -226,35 +223,32 @@ Settings:
     Select Import Type: Select Import Type
     Select Export Type: Select Export Type
     Import Subscriptions: Import Subscriptions
-    Import FreeTube: Import FreeTube
-    Import YouTube: Import YouTube
-    Import NewPipe: Import NewPipe
+    Import FreeTube: FreeTube Import
+    Import YouTube: YouTube Import
+    Import NewPipe: NewPipe Import
     Check for Legacy Subscriptions: Check for Legacy Subscriptions
     Export Subscriptions: Export Subscriptions
     Export FreeTube: Export FreeTube
     Export YouTube: Export YouTube
-    Export NewPipe: Export NewPipe
+    Export NewPipe: NewPipe Export
     Import History: Import History
     Export History: Export History
     Profile object has insufficient data, skipping item: Profile object has insufficient
-      data, skipping item
-    All subscriptions and profiles have been successfully imported: All subscriptions
-      and profiles have been successfully imported
+  data, skipping item
+    All subscriptions and profiles have been successfully imported: Imported all
+  subscriptions and profiles
     All subscriptions have been successfully imported: All subscriptions have been
-      successfully imported
+  imported
```

Comment:
The successfully can be moved behind the imported, but as before, I would leave it in. Imo, it just makes the text feel more positive from a users perspective

Reply:
It makes for more text to read.

Reply:
As I said it is debatable and my opinion. If some more people would like to add to that or discuss it and the majority says one way or the other I am open to leaving your change as it is. But these text changes are always dependent on what people want

Reply:
It is never a problem to remove it, because it never does anything useful.